notebooks/format_benchmark.py

```diff
@@ -0,0 +1,169 @@
+import os
+import re
+import simplejson as json
+from dateutil import parser
+
+
+def convert_benchmark(output_dir, monitoring_log):
+    """
+    Converts and Saves a downloaded monitoring log to a formated JSON
+    :param output_dir: the desired output directory to store the monitoring log, plots and json file
+    :param monitoring_log: monitoring log downloaded from the googgle bucket (JSON)
+    """
+
+    with open(monitoring_log, "r") as bm:
+
+        # read json in a dict and
+        call_logs = json.load(bm)
+
+    # for each task call in dict
+    converted_call_logs = {}
+
+    for name, info in call_logs.items():
+
+        # summary
+        num_of_cores = None
+        total_mem = {"Size": None, "Unit": None}
```

Comment:
Could you make the field names in these dictionaries either lowerCamelCase or lower_snake_case? No preference for one over the other, they're just the usual conventions for JSON.